connector/signaltometricsconnector/metadata.yaml

```diff
@@ -0,0 +1,32 @@
+type: signaltometrics
+scope_name: github.com/elastic/opentelemetry-collector-components/connector/signaltometricsprocessor
+
+status:
+  class: connector
+  stability:
+    alpha: [traces_to_metrics, logs_to_metrics, metrics_to_metrics]
+  distributions: []
+
+tests:
+  config:
+    spans:
+      - name: span.duration.histogram
+        description: "Span duration as histogram"
+        unit: ms
+        histogram:
+          value: Microseconds(end_time - start_time)
+      - name: span.duration.exponential_histogram
+        description: "Span duration as exponential histogram"
+        unit: ms
+        exponential_histogram:
+          value: Microseconds(end_time - start_time)
```

Comment:
```suggestion
      - name: span.duration.histogram
        description: "Span duration as histogram"
        unit: micros
        histogram:
          value: Microseconds(end_time - start_time)
      - name: span.duration.exponential_histogram
        description: "Span duration as exponential histogram"
        unit: micros
        exponential_histogram:
          value: Microseconds(end_time - start_time)
```

Reply:
Ah, thanks for catching this. Should it be `micros` or `us`? Do we have the units defined somewhere?

Reply:
You're right, it should be `us`. Units are defined by UCUM. See also https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units.

Reply:
Fixed in https://github.com/elastic/opentelemetry-collector-components/pull/169/commits/fe5438de6c7c3431720d55edd301d5a28cc25bbe